supported/osg-htc/osg-hosted-ce/Chart.yaml

```diff
@@ -3,4 +3,4 @@ apiVersion: v1
 appVersion: "V5-branch"
 description: OSG Hosted Compute Entrypoint
 name: osg-hosted-ce
-version: 4.6.3
+version: 4.6.4
```

Comment:
There's enough stuff getting ripped out that we should probably bump the minor version. We roughly try to follow semver https://semver.org/#summary

Reply:
Bumped to 4.7.0